### PR TITLE
addCardList

### PR DESCRIPTION
**Pull request by amirhosein0980**
addCaardList and addCardjson 
Please check the files to merge

### CODE REVIEW

File: src/components/widgets/Cardjson.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
+[
```
Comment (chalist) [inline src/components/widgets/Cardjson.json:1]: Move and rename this file to new folder in src: `src/data/Card.json`
other guys can create their own files there too.

File: src/components/widgets/CardList.jsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import DataCard from '@/components/widgets/Cardjson.json';
+
+function CardList() {
+  return (
+    <>
+      <div className="w-4/5 h-80 m-auto mt-2.5 flex justify-between">
+        {DataCard[0].data.items.map((item) => (
+          <div className="w-60 h-full">
+            <div className="w-full h-3/6 bg-yellow-500 rounded-tr-2xl rounded-tl-2xl overflow-x-scroll">{item.image_src}</div>
+            <div className="w-full h-3/6 rounded-br-2xl rounded-bl-2xl" style={{ backgroundColor: 'rgb(247,247,247)' }}>
```
Comment (chalist) [inline src/components/widgets/CardList.jsx:10]: remove stype attr and use official color preset of Tailwind classes. something like these: `bg-orange-300` or `bg-yellow-400`.

File: src/components/widgets/CardList.jsx [hunk 3]
```diff
@@ -0,0 +1,21 @@
+import DataCard from '@/components/widgets/Cardjson.json';
+
+function CardList() {
+  return (
+    <>
+      <div className="w-4/5 h-80 m-auto mt-2.5 flex justify-between">
+        {DataCard[0].data.items.map((item) => (
+          <div className="w-60 h-full">
+            <div className="w-full h-3/6 bg-yellow-500 rounded-tr-2xl rounded-tl-2xl overflow-x-scroll">{item.image_src}</div>
```
Comment (chalist) [inline src/components/widgets/CardList.jsx:9]: Why you didn't use `img` tag with `image_src`?

File: src/components/widgets/Cardjson.json (hunk 1 above)
Comment (amirhosein0980) [reply src/components/widgets/Cardjson.json]: Edited

File: src/components/widgets/CardList.jsx (hunk 2 above)
Comment (amirhosein0980) [reply src/components/widgets/CardList.jsx]: Edited

File: src/components/widgets/CardList.jsx (hunk 3 above)
Comment (amirhosein0980) [reply src/components/widgets/CardList.jsx]: Edited

**Comment by chalist** [comment]
@amirhosein0980 Check PR conflicts and fix it, please.

**Comment by amirhosein0980** [comment]
@chalist Edited PR conflicts

**Pull request merged**